csfieldguide/chapters/content/en/human-computer-interaction/sections/user-experience.md

```diff
@@ -0,0 +1,559 @@
+
+{panel type="teacher-note"}
+
+# Affinity Diagram Tips
+
+This is a very small affinity diagram.
+Your students should have a lot of data to work with from their surveys and therefore should have at least 20 sticky notes that they can sort into 4 or 5 strong themes.
+
+{panel end}
+
+### Step 4: Describe your user.
+
+**UX Methods: Empathy Map, User Persona and User Stories**
+
+Now that we’ve established who our users are and what they want, it would be useful to create a *User Persona*.
+A *User Persona* is a character that is representative of your target user.
+To help us with creating a user persona, we will use an *Empathy Map* to give us a structure to work with.
+An *Empathy Map* defines four aspects of the persona: how they Think and Feel and what they Do and Say, and is drawn in quadrants.
+We’ve created a very basic one below to help us develop a persona for the target user of our sleep habits app:
+
+|                                                                                                                                          	| **User**                                                                                                                                                                                                                    	|
+|------------------------------------------------------------------------------------------------------------------------------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| **Says**<br>Things the user says out loud in an interview or wrote in their survey.<br><br>*I am so tired.<br>I think I stayed up too late last night.* 	| **Thinks**<br>What do they think about? What are their needs, goals and motivations?<br><br>*I wish I could get more sleep.<br>Maybe I should go to bed earlier.<br>How many hours of sleep do I actually need?<br>Why am I always tired?* 	|
+| **Does**<br>What actions does the user do?<br><br>*Always hits “snooze” on the alarm.<br>Drinks coffee first thing in the morning.*                     	| **Feels**<br>Consider the tone they use in an interview and/or survey answers.<br><br>*Tired.<br>Annoyed.<br>Frustrated.*                                                                                                                	|
+
+Using our Empathy Map we are able to derive the target users needs and from here we created a persona representative of our target user.
+Meet Bevan:
+
+**Bevan, 22, Accounting Student at University**
+
+{image file-path="img/chapters/bevan.jpg" alt="Photo of Bevan for persona."}
+
+*__About__*
+
+*Bevan is a busy undergraduate student.
```

Comment:
Could this section be gender neutral to make it more inclusive?

Reply:
I disagree. Each persona needs to be a well-defined hypothetical person (singular) and being PC about it detracts from the information it needs to present

Reply:
Made it a man because the character needs to be clearly defined. The case studies I added in the useful links/further reading section all use women, so this felt like the best way to balance it.

Reply:
Referring to a person using they/them pronouns _is_ still clearly defining a character, people do identify as they/them. It would be best if we could have multiple user stories that would be presented in a slideshow sort of format (where you can swipe through them). These different user stories would show users from different cultures/genders etc...

Reply:
While I like that idea, we need to remember that we're teaching Computer Science here. Adding more of the same example from slightly different perspectives adds almost nothing with respect to education and shouldn't be a priority

Reply:
Had a great discussion with @timbellnz and @caitlind about this morning, here's a summary:
- We should be as inclusive as possible, where possible.
- If we are using one person, it would be better to be inclusive with a 'they' pronoun.
- However it would be better to have multiple personas, for example:
  - Female, Nurse, trying to get better sleep around shifts.
  - Male, Student, sleep for exams.
  - Neutral, Author and wheelchair bound, sleep to improve health.
- Rather than having a long list down the page, a slideshow that you can look through (as @courtneycb suggested) would be ideal.
- When the students do their own user studies they do need to be precise with definitions of their users where appropriate (as @hayleyavw @eAlasdair mentioned) so we should encourage them to that. However as an example for what you should do, we should be inclusive and broad in our suggestions.
- An addition of a panel stating the above point with examples would be a great addition. For example:
  - "*The three examples below are possible users for this application. When performing your own user studies, you should list and research their gender, age, culture, etc. For example: Elderly people across different countries behave differently.*"
- These changes would keep the 'science' intact, while being inclusive to all, in a field which definitely needs to be more inclusive.
- The photo should either be replaced with an icon or removed completely, as when a student sees the image, they immediately create a mental picture of this person based off their experiences. We would like them to be subjective in this situation.

Reply:
I agree, but it's not something for this PR. I'll make an issue

Reply:
#992

Reply:
Agree that it should be in a seperate PR but this point should be addressed in this PR:

> If we are using one person, it would be better to be inclusive with a 'they' pronoun.

Reply:
Again, it needs to be a good example first and foremost; while some people do refer to themselves as they/them etc putting that in an example persona (I'd be fine if this wasn't a persona) gives the reader more ambiguity in what they should be doing

> When the students do their own user studies they do need to be precise with definitions of their users where appropriate